src/hooks/useEnableAnalyticsForm.tsx

```diff
@@ -0,0 +1,19 @@
+import { useState } from 'react';
+
+export type UseEnableAnalyticsForm = {
+  enableSaveActions: boolean;
+  updateSaveActions: (enable: boolean) => void;
+};
+
+export const useEnableAnalyticsForm = (): UseEnableAnalyticsForm => {
```

Comment:
I'm not against hooks, but this one look especially short?

Reply:
Yes I will set the state in the sign up.